src/components/message/edit-message-actions/styles.scss

```diff
@@ -15,3 +14,14 @@
     color: theme.$color-greyscale-12;
   }
 }
+
+.zui-tooltip__content {
```

Comment:
I don't think we should be overriding zui component styles. This could cause every tooltip in the whole app (even all the zapps) to take on these properties.

Reply:
Okay makes sense. I'll need to go into zUI and make some changes for these

Reply:
Are these what we want _all_ tooltips to look like? If so, we can just delete these from here and merge this and then update zUI as a follow up.

Reply:
This is the component in the design system - https://www.figma.com/file/wTOVNZonl9MmEzORwBHDuq/zDS-Web-%26-Global?type=design&node-id=1641-3344&mode=design&t=TXIBEvHDcJ0gmcZN-4. 

I think this would mean updating zUI. However, I can remove these overriding styles for now and just use what zUI provides for the time being